Github Workflows For Linux and Athena Are Failing

Although this appears to be valid code that does build, both the linux and athena workflows continue to fail.

Here is output:

Run git config --global --add safe.directory /__w/vendor-template/vendor-template
  git config --global --add safe.directory /__w/vendor-template/vendor-template
  git fetch --prune --unshallow
  shell: sh -e {0}
fatal: detected dubious ownership in repository at '/__w/NFR-Lib/NFR-Lib'
To add an exception for this directory, call:

	git config --global --add safe.directory /__w/NFR-Lib/NFR-Lib
Error: Process completed with exit code 1[2](https://github.com/NorthernForce/NFR-Lib/actions/runs/5116689783/jobs/9199094154#step:4:2)[8](https://github.com/NorthernForce/NFR-Lib/actions/runs/5116689783/jobs/9199094154#step:4:9).

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -29,7 +29,7 @@ jobs:
           submodules: true
       - name: Fetch all history and metadata
         run: |
-          git config --global --add safe.directory /__w/vendor-template/vendor-template
+          git config --global --add safe.directory /__w/NFR-Lib/NFR-Lib
           git fetch --prune --unshallow
       - name: Build with Gradle
         run: ./gradlew build --max-workers 1 ${{ matrix.build-options }}

